File: main/webapp/js/service/repairService.js
app.service("repairService",function($http){
	
	//查询报修设备列表
	this.search = function(pages,rows,repairVo){
		return $http.post("repair/search.do?pages="+pages+"&rows="+rows,repairVo);
	}
	

	//报废
	this.scrap = function(scrap){
		return $http.post("repair/scrap.do",scrap);
	}
	
	//修理成功处理
	this.repairSuccess = function(repair){
		return $http.post("repair/repairSuccess.do",repair);
	}
	
})

File: main/java/com/laboManager/service/RepairService.java
package com.laboManager.service;


import com.laboManager.entity.PageResult;
import com.laboManager.entity.RepairVo;
import com.laboManager.pojo.TbRepair;
import com.laboManager.pojo.TbScrap;

public interface RepairService {
	/**
	 * 查询报修的设备
	 * @param pages
	 * @param rows
	 * @param repair
	 * @return
	 */
	public PageResult search(int pages,int rows,RepairVo repair);
	
	/**
	 * 报废
	 * @param scrap
	 */
	public void scrap(TbScrap scrap);
	
	/**
	 * 修理成功处理
	 */
	
	public void repairSuccess(TbRepair repair);
}
